docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -110,39 +110,47 @@ _Available in public preview in Grafana Open Source and Enterprise_
 
 You can now add buttons to your Canvas visualizations. Buttons can be configured to call an API endpoint. This pushes Grafana's capabilities to new heights, allowing you to create interactive dashboards that can be used to control external systems.
 
-To learn more, refer to our [Canvas button element documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/canvas/#TODO).
+To learn more, refer to our [Canvas button element documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/canvas/#button).
 
-TODO: Add image / gif / video
+{{< video-embed src="/media/docs/grafana/screen-recording-10-2-canvas-button-element-demo.mp4" max-width="750px" caption="Canvas button element demo" >}}
 
-### Time series visualization now support y-axis zooming
+### Y-axis zoom support in time series and candlestick visualizations
 
-_Available in public preview in Grafana Open Source and Enterprise_
+_Generally available in Grafana Open Source and Enterprise_
 
 <!-- Nathan Marrs -->
 <!-- Cloud -->
 
-You can now zoom in on the y-axis of your time series visualizations. This is useful when you want to focus on a specific range of values.
+You can now zoom in on the y-axis of your time series and candlestick visualizations. This is useful when you want to focus on a specific range of values. To zoom in on the y-axis, hold the shift key while clicking and dragging on supported visualizations. Double click to reset the zoom.
 
-To learn more, refer to our [Time series documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/time-series/#TODO).
```

Comment:
investigated and it appears we don't currently have any documentation related to zoom feature 😬